src/SFA.DAS.Commitments.Database/Views/CommitmentSummary.sql

```diff
@@ -30,7 +30,18 @@ AS (SELECT Id, CommitmentId, AgreementStatus,
 	FROM 
 		Apprenticeship a
 )
-SELECT c.*, 
+SELECT
+	c.Id, c.Reference, c.EmployerAccountId,
+	ale.LegalEntityId as 'LegalEntityId', ale.[Name] as 'LegalEntityName', c.ProviderId, p.[Name] as 'ProviderName',
+	c.CommitmentStatus, c.EditStatus, c.CreatedOn, c.LastAction, 
+	c.LastUpdatedByEmployerEmail, c.LastUpdatedByProviderEmail, c.LastUpdatedByEmployerName, c.LastUpdatedByProviderName,
+	ale.[Address] as 'LegalEntityAddress',
+	ale.[OrganisationType] as 'LegalEntityOrganisationType',
+	c.TransferSenderId,
+	ts.[Name] as 'TransferSenderName',
+	c.TransferApprovalStatus, c.TransferApprovalActionedByEmployerEmail,
+	c.TransferApprovalActionedByEmployerName, c.TransferApprovalActionedOn, ale.PublicHashedId as 'AccountLegalEntityPublicHashedId', c.Originator, c.ApprenticeshipEmployerTypeOnApproval,
+	c.IsFullApprovalProcessed, c.IsDeleted, c.AccountLegalEntityId, c.IsDraft, c.WithParty, c.RowVersion, c.LastUpdatedOn, c.Approvals, c.EmployerAndProviderApprovedOn,
```

Comment:
At last !!!!! I hated c.*. I'm sure we don't need all those fields, but considering that this view is going away in the near future it doesn't seem worth putting anymore effort than this into it